polkadot-protocol/protocol-components/accounts/address-formats.md

```diff
@@ -0,0 +1,183 @@
+
+Currently, the valid values are:
+
+- `00000000b..=00111111b` (0 to 63 inclusive)
+
+    Simple account/address/network identifier. The byte can be interpreted directly as such an identifier.
+
+- `01000000b..=01111111b` (64 to 127 inclusive)
+
+    Full address/address/network identifier. The lower 6 bits of this byte represent the upper 6 bits of a 14-bit identifier, while the next byte defines the lower 8 bits. This allows for identifiers ranging from 0 to 16,383 (2²¹).
+
+- `10000000b..=11111111b` (128 to 255 inclusive)
+
+    This section is reserved for future address format extensions. The address type 42 is designed to be valid across all Polkadot SDK networks that support fixed-length addresses. 
+
+    However, for production networks, using a network-specific version may be preferable to prevent key reuse between networks and mitigate associated issues. 
+
+    By default, Polkadot SDK nodes display keys in address type 42, but Polkadot SDK-based chains with different node implementations—such as those in the Polkadot ecosystem—might use a different default address type.
+
+## Address Length in bytes
+
+There are 16 different address formats, each identified by the total payload length in bytes, which includes the checksum.
+
+| Total | Type | Raw account | Checksum |
+|-------|------|-------------|----------|
+| 3     | 1    | 1           | 1        |
+| 4     | 1    | 2           | 1        |
+| 5     | 1    | 2           | 2        |
+| 6     | 1    | 4           | 1        |
+| 7     | 1    | 4           | 2        |
+| 8     | 1    | 4           | 3        |
+| 9     | 1    | 4           | 4        |
+| 10    | 1    | 8           | 1        |
+| 11    | 1    | 8           | 2        |
+| 12    | 1    | 8           | 3        |
+| 13    | 1    | 8           | 4        |
+| 14    | 1    | 8           | 5        |
+| 15    | 1    | 8           | 6        |
+| 16    | 1    | 8           | 7        |
+| 17    | 1    | 8           | 8        |
+| 35    | 1    | 32          | 2        |
+
+## Checksum Types
+
+Polkadot SDK offers several potential checksum strategies, each providing varying length and longevity guarantees. 
+
+There are two types of checksum preimages (`SS58` and `AccountID`) along with a range of checksum lengths from 1 to 8 bytes.
+
+In all cases for Polkadot SDK, the Blake2b-512 hash function is used. The variants simply select the preimage used as the input to the hash function and the number of bytes taken from its output.
+
+The bytes used are always the left-most bytes. The input consists of the non-checksum portion of the SS58 byte series, which is used as input to the base-58 function. 
+
+For example, this input is formed by concatenating the `concat ( <address-type>, <address> )`. A context prefix of `0x53533538505245` (the string "SS58PRE") is then prepended to this input to create the final hashing preimage.
```

Comment:
For some of these functions, again, it would be nice to link them for clarity.

Reply:
ditto